File: Morning Snack/Python/common_denominator.py
user_input_one = int(input("Enter a number: "))
user_input_two = int(input("Enter a number: "))
highestcommon = 0;

for count in range (1, user_input_one and user_input_two):
    if(user_input_one % count == 0 and user_input_two % count == 0):        
        highestcommon = count;

print(highestcommon);   

File: Morning Snack/Java/CommonDenominator.java
import java.util.Scanner;
public class CommonDenominator{
    public static void main(String[]args){
    Scanner input = new Scanner(System.in);

    System.out.print("Enter a number: ");
    int userInputOne = input.nextInt();
    System.out.print("Enter a number: ");
    int userInputTwo = input.nextInt();
    
    int highestcommon = 0;
    int count;

for(count = 1; count <= userInputOne && count <= userInputTwo; count++){
    if(userInputOne % count == 0 && userInputTwo % count == 0){
        
                highestcommon = count;

        }
      }
         System.out.println(highestcommon);
    }
}
